packages/uni-virtualizer/src/lib/VirtualScroller.ts

```diff
@@ -436,6 +455,8 @@ export class VirtualScroller extends VirtualRepeater {
       top = this._containerElement.scrollTop;
     } else {
       const containerBounds = this._containerElement.getBoundingClientRect();
+      console.log("calced", containerBounds.top + scrollY)
+      console.log("offsetTop", (this._containerElement as HTMLElement).offsetTop)
```

Comment:
Looks like these log statements need to be removed.

Reply:
🤦‍♂️How'd I miss it!